blocks/cards/cards.js

```diff
@@ -1,16 +1,39 @@
 import { createOptimizedPicture } from '../../scripts/aem.js';
+import { domEl } from '../../scripts/dom-helpers.js';
 
 export default function decorate(block) {
   block.classList.add('calcite-mode-dark');
 
+  const processSimpleCard = function (div) {
+    if (!block.classList.contains('simple')) {
+      return;
+    }
+    const anchorEl = div.querySelector('a');
+    if (anchorEl) {
+      anchorEl.textContent = '';
+      div.append(anchorEl);
+      div.querySelector('.button-container').remove();
+    }
+    const cardBodyContent = domEl('div', { class: 'card-body-content' });
+    if (anchorEl) {
+      anchorEl.append(cardBodyContent);
+    } else {
+      div.append(cardBodyContent);
+    }
```

Comment:
I added this in another thread, but the comment got lost as "outdated", how about unifying the "anchorEl" conditional to improve readability? Maybe something like this:
```suggestion
    const cardBodyContent = domEl('div', { class: 'card-body-content' });

    const anchorEl = div.querySelector('a');
    if (anchorEl) {
      anchorEl.replaceChildren(cardBodyContent);
      div.append(anchorEl);
      div.querySelector('.button-container').remove();      
    } else {
      div.append(cardBodyContent);
    }
```

Reply:
Missed the other thread. Yes I have unified the anchor el if condition. Updated